src/modules/extendedCallWindow/i18n/en_GB.js

```diff
@@ -79,6 +79,10 @@ module.exports = {
                 0, 1, 2, 3, 4, 6, 7, 14, 15, 16, 17, 18, 23, 26, 35, 36, 37, 38,
                 39, 40,
             ],
+            [/^more firefighter$/u]: [
+                0, 1, 2, 3, 4, 6, 7, 14, 15, 16, 17, 18, 23, 26, 35, 36, 37, 38,
+                39, 40,
+            ],
```

Comment:
use `[/^more firefighters?$/u]: [` in line 78 instead of adding these 4 lines.

Reply:
Sorry I dont understand what your asking, you want me to condense it to one line?

Reply:
There already is a list for plural. You can add singluar by changing the regex (key). That is cleaner and easier to maintain than adding singular as an extra list.

Reply:
`^more firefighter(s?)$/u` should do the job